docs/source/tutorial.md

```diff
@@ -1,24 +1,43 @@
-# Tutorial
+# Tutorials
 
-This page covers brief tutorials and notes on how to use CytoTable.
+Start here if you are new to CytoTable. We’ve split material by audience:
 
-## CellProfiler CSV Output to Parquet
+- **Image analysts (no engineering background required):** follow the narrative tutorials below. They include downloadable data, exact commands, and what to expect.
+- **Engineers / power users:** see the Software Engineering Guide for tuning and integration details, or use the quick recipe below.
 
-[CellProfiler](https://cellprofiler.org/) pipelines or projects may produce various CSV-based compartment output (for example, "Cells.csv", "Cytoplasm.csv", etc.).
-CytoTable converts this data to Parquet from local or object-storage based locations.
+```{admonition} Who this helps (and doesn’t)
+- Helps: image analysts who want to get CellProfiler/NPZ outputs into Parquet with minimal coding; people comfortable running a few commands.
+- Not ideal: raw image ingestion or pipeline authoring (use CellProfiler/DeepProfiler upstream); workflows needing a GUI-only experience.
+- Effort: install, copy/paste a few commands, validate outputs in minutes.
+```
+
+```{toctree}
+---
+maxdepth: 2
+caption: Tutorials (start here)
+---
+tutorials/cellprofiler_sqlite_to_parquet
+tutorials/npz_embeddings_to_parquet
+tutorials/multi_plate_merge_tablenumber
+software_engineering
+```
+
+Looking for variations or troubleshooting? See the Software Engineering Guide.
+
+## Quick recipe: CellProfiler CSV to Parquet
+
+This short recipe is for people comfortable with Python/CLI and parallels our older tutorial. If you prefer a guided, narrative walkthrough with downloadable inputs and expected outputs, use the tutorial above.
```

Comment:
```suggestion
This short recipe is for people comfortable with Python/CLI and parallels our older tutorial. 
If you prefer a guided, narrative walkthrough with downloadable inputs and expected outputs, use the tutorial above.
```
Recommend double-checking that there is one sentence per line.

Reply:
Thanks will do!